### PR TITLE
Pqsignature add pastel verify sign

### PR DESCRIPTION
**Pull request by Dicklesworthstone**

### CODE REVIEW

**Review by mateeullahmalik — DISMISSED** [review]

**Review by a-ok123 — APPROVED** [review]

**Review by mateeullahmalik — DISMISSED** [review]

File: common/b85/b85.go [hunk 1]
```diff
@@ -0,0 +1,152 @@
+package b85
```
Comment (nguyen202) [inline common/b85/b85.go:1]: could you add unit tests for this package?

Comment (mateeullahmalik) [reply common/b85/b85.go:1]: @nguyen202 will add in next PR

**Review by nguyen202 — DISMISSED** [review]
LGTM!

**Review by nguyen202 — APPROVED** [review]
LGTM!

**Review by a-ok123 — APPROVED** [review]

**Pull request merged**